samples/snippets/create_conversation_with_ttl.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# [START contactcenterinsights_create_conversation_with_ttl]
+from google.cloud import contact_center_insights_v1
+from google.protobuf import duration_pb2
+
+
+def create_conversation_with_ttl(
+    project_id: str,
+    transcript_uri: str = "gs://cloud-samples-data/ccai/chat_sample.json",
+    audio_uri: str = "gs://cloud-samples-data/ccai/voice_6912.txt",
+) -> contact_center_insights_v1.Conversation:
+    # Construct a parent resource.
+    parent = contact_center_insights_v1.ContactCenterInsightsClient.common_location_path(
+        project_id, "us-central1"
```

Comment:
Nit: consider making the location an arg to the function as well.

Reply:
Thank you for the suggestion! We'd prefer hard-coding the location here because we intend to have this sample be very similar to the existing [create_conversation](https://github.com/googleapis/python-contact-center-insights/blob/main/samples/snippets/create_conversation.py) sample since they're gonna be close to each other in the Insights public docs. The main difference between the existing sample and this one is that this sample sets a TTL on the conversation. Let me know what you think!